components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add.jsp

```diff
@@ -245,6 +245,7 @@
                         $(jQuery('#encryption_method_row')).hide();
                         $(jQuery('#encryption_algorithm_row')).hide();
                         $(jQuery('#callback_row')).show();
+                        $(jQuery('#bypass_client_credentials').hide());
 
                     } else if(oauthVersion == "<%=OAuthConstants.OAuthVersions.VERSION_2%>") {
```

Comment:
Shall we fix the indentation issue ?

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1058